tests/test_client.py

```
@@ -62,395 +51,528 @@ def test_load_s3(get_dataset_from_s3, dataset):
+            # *** Add a check here to see the actual forecast_reference_time values ***
+            if "forecast_reference_time" in ds.coords:
+                print(
+                    f"Forecast reference times in loaded data: {ds['forecast_reference_time'].values}"
+                )
+            else:
+                print("No 'forecast_reference_time' coordinate found in loaded data.")
+            return ds
+    except Exception as e:
+        raise ValueError(
+            f"Failed to load test Zarr from {zip_path} for dataset '{dataset_name}': {type(e).__name__} - {e}"
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)